core/lib/generators/workarea/install/install_generator.rb

```diff
@@ -22,10 +20,10 @@ def add_requires
     end
 
     def mount_routes
-      route "mount Workarea::Storefront::Engine => '/', as: 'storefront'"
+      route "mount Workarea::Core::Engine => '/'"
       route "mount Workarea::Admin::Engine => '/admin', as: 'admin'"
       route "mount Workarea::Api::Engine => '/api', as: 'api'" if Workarea::Plugin.installed?(:api)
-      route "mount Workarea::Core::Engine => '/'"
+      route "mount Workarea::Storefront::Engine => '/', as: 'storefront'"
```

Comment:
For the redirect and errors functionality to kick in, storefront always needs to be last.